pos_fiscal_position/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+The main purpose of this module is to avoid configuration errors related to fiscal positions and taxes by the final user.
+The PoS fiscal position button can be hidden to delegate this task to the PoS settings.
```

Comment:
hi, not sure to understand errors related to fiscal position. Could you provide an exemple ?